src/test/java/com/hm/achievement/db/SQLiteDatabaseBasicTest.java

```diff
@@ -0,0 +1,228 @@
+package com.hm.achievement.db;
+
+import com.hm.achievement.AdvancedAchievements;
+import com.hm.achievement.exception.PluginLoadError;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+import utilities.MockUtility;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+
+/**
+ * Class for testing SQLite Database.
+ *
+ * @author Rsl1122
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class SQLiteDatabaseBasicTest extends SQLiteDatabaseTest {
+
+    @Before
+    public void setUp() throws Exception {
+        MockUtility mockUtility = MockUtility.setUp()
+                .mockLogger()
+                .mockPluginConfig();
+        AdvancedAchievements pluginMock = mockUtility.getPluginMock();
+
+        db = new SQLiteDatabaseManager(pluginMock);
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        if (db != null) {
+            db.shutdown();
+        }
+    }
+
+    @Test
+    public void testInitialization() throws PluginLoadError {
+        initDB();
```

Comment:
This is a heavy method call. Underneath the hoods we are creating a whole new database file with close to 50 database tables at the beginning of each test. On my old laptop equipped with a hard disk drive, the unit tests take over a minute to complete.

What do you think about moving this to a preceding `@BeforeClass` method or if not possible, avoid systematically deleting the database?  We could then simply clear the `achievements` table at the beginning of each test. This should speed them up quite a bit! 😉

Reply:
When testing on my own plugin at one point `@BeforeClass` was called like `@Before`

I can't remember what caused it so this is still an open option

Reply:
Done, improved test time drastically.

Reply:
Indeed, the tests are much faster now!